package.json

```
@@ -234,6 +234,11 @@
                     "type": "string",
                     "default": null,
                     "description": "Image prefix for docker images ie 'docker.io/brendanburns'"
+                },
+                "vsminikube.checkForUpgrade": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Check if update is available for minikube?"
```

Suggest minor wording tweak:
```suggestion
                    "description": "Notify on startup if update is available for minikube"
```